### PR TITLE
Fix for past weekends series slug

### PR DESCRIPTION
**Pull request by jasoncallibrity**
added logic for capturing series slug from the videos (read as message) they contain

### CODE REVIEW

File: live.html [hunk 1]
```diff
@@ -267,14 +267,18 @@ <h3 class="collection-header push-bottom">
   <div class="row">
     <div class="card-deck carousel" data-crds-carousel="mobile-scroll">
       <div class="feature-cards card-deck--expanded-layout">
-        {% assign recent_messages = site.messages | sort: 'date' | reverse | slice: 0,4 %} {% for message in recent_messages %}
+        {% assign recent_messages = site.messages | sort: 'date' | reverse | slice: 0,4 %} 
+        {% for message in recent_messages %}
+        {% for series in site.series %}
+          {% assign series_slug = series.slug | where: "videos.slug", message.slug %}
```
[the remark this answers is not in the document]
Comment (agause) [reply live.html]: @jasoncallibrity Can we give a message the ability to know the series to which it belongs?

File: live.html [hunk 2]
```diff
@@ -267,14 +267,18 @@ <h3 class="collection-header push-bottom">
   <div class="row">
     <div class="card-deck carousel" data-crds-carousel="mobile-scroll">
       <div class="feature-cards card-deck--expanded-layout">
-        {% assign recent_messages = site.messages | sort: 'date' | reverse | slice: 0,4 %} {% for message in recent_messages %}
+        {% assign recent_messages = site.messages | sort: 'date' | reverse | slice: 0,4 %} 
+        {% for message in recent_messages %}
+        {% for series in site.series %}
```
Comment (lukemoderwell) [inline live.html]: Not sure that you need this extra loop. You could do the `assign` for a series and then set line `276` to `series.slug`.

Comment (lukemoderwell) [reply live.html]: Looks like this extra loop may be messing up some stuff. Deploy preview looks like it has the same image for all 4 messages.

Comment (jasoncallibrity) [reply live.html]: Nah, they literally had the same image in contentful. I mixed it up

Comment (jasoncallibrity) [reply live.html]: I'm using the slug from the message to find the series it belongs to. I have to loop through the series and compare the message slug to the videos slug until I get a match. Unless I'm missing something?

File: live.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (jasoncallibrity) [reply live.html]: I can add  {% break %} to it as I discovered it while we were talking to this existing code.

Do your second point, I will do some digging tomorrow to see if this is best practice. To be honest I'm not 100% sure. Thanks for calling it out!

**Pull request closed without merging**